chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -278,10 +278,10 @@ sync
 <% else -%>
   <% if @target_platform_version == "11.3" -%>
       <package>suse-sle11-openstack-cloud-release</package>
```

Comment:
Well, there's the idea of potentially getting 11.4 compute nodes later on, and having the code here would be a reminder to update it. But again, I'm fine removing it too :-)